From 4f15d2234608e82159d030dadb17af678cfad626 Mon Sep 17 00:00:00 2001
From: Ian Lance Taylor <iant@golang.org>
Date: Tue, 12 Jul 2022 19:42:43 -0700
Subject: [PATCH] libgo: don't include <linux/fs.h> when building
 gen-sysinfo.go

Removing this doesn't change anything at least with glibc 2.33.
The include was added in https://go.dev/cl/6100049 but it's not
clear why.

Fixes PR go/106266

Reviewed-on: https://go-review.googlesource.com/c/gofrontend/+/417294
(cherry picked from commit f35d65517a59565758107c5b1a51a5fa382f8d1a)
---
 libgo/sysinfo.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/libgo/sysinfo.c b/libgo/sysinfo.c
index 8ce061e2f..336a59837 100644
--- a/libgo/sysinfo.c
+++ b/libgo/sysinfo.c
@@ -158,9 +158,6 @@
 #if defined(HAVE_LINUX_ETHER_H)
 #include <linux/ether.h>
 #endif
-#if defined(HAVE_LINUX_FS_H)
-#include <linux/fs.h>
-#endif
 #if defined(HAVE_LINUX_REBOOT_H)
 #include <linux/reboot.h>
 #endif
-- 
2.37.1

